src/actions/walletActions.js

```diff
@@ -155,16 +156,22 @@ export function autoSelectCoins() {
   };
 }
 
-export function spendSlices(inputs, changeSlice) {
+/**
+ * @description Given the state of the transaction store, check status of
+ * the slices being used in the spend tx and update them once changed state is confirmed.
+ * @param {Object} changeSlice - the slice that was chosen as the change output
+ */
+export function updateTxSlices(changeSlice, retries = 5) {
```

Comment:
start at 5 retries left. This gets counted down when the function is called recursively.